src/sentry/static/sentry/app/views/settings/project/projectFilters/index.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import {t} from '../../../../locale';
+import GroupTombstones from './groupTombstones';
+import ProjectFiltersChart from './projectFiltersChart';
+import ProjectFiltersSettings from './projectFiltersSettings';
+import SentryTypes from '../../../../proptypes';
+import SettingsPageHeader from '../../components/settingsPageHeader';
+import TextBlock from '../../components/text/textBlock';
+
+class ProjectFilters extends React.Component {
+  static contextTypes = {
+    organization: SentryTypes.Organization,
+    project: SentryTypes.Project,
+  };
+
+  constructor(...args) {
+    super(...args);
+    this.state = {
+      activeSection: 'data-filters',
+    };
+  }
+
+  setProjectNavSection = section => {
```

Comment:
I'm guessing this is a holdover from earlier behavior, but we should try to manage subnavs via the router / make them declared routes.